docs/src/pages/style/links/links.md

```diff
@@ -0,0 +1,38 @@
+---
+components: Link
+---
+
+# Links
+
+<p class="description">The Link component creates a hyperlink to other web pages, files, locations within the same page, email addresses, or any other URL.</p>
+
```

Comment:
What I meant was, it might be helpful to explain why a developer should use the Link component rather than simply using an HTML anchor. I.e. what is unique the purpose of this component?

Reply:
I'll change it to that, I just thought this was a better description than what was there